windows/src/test/unit-tests/Makefile

```diff
@@ -18,6 +13,16 @@ test:
 
 # ----------------------------------------------------------------------
 
+# TODO: move shared-tests to /common
```

Comment:
Related note:  I noticed that a lot of keyboard testing resources are getting shuffled around... but to subfolders of `developer`.  I suppose this note is also with respect to those file movements?

Reply:
The keyboard testing resources here are **compiler** tests, not **keyboard** tests. Therefore, those resources belong in the developer unit tests folder (where they will be moved in a later PR). There are other unit tests that are testing shared code, such as those moved in #6758.